src/store/DevfileRegistries/index.ts

```diff
@@ -171,11 +171,15 @@ export const actionCreators: ActionCreators = {
         const patchedJSONString = JSON.stringify(schemav1).replaceAll('#/definitions', '#/oneOf/0/definitions');
         const parsedSchemaV1 = JSON.parse(patchedJSONString);
```

Comment:
```parsedSchemaV1``` => ```schemav100```

To be consistent with  
```
            ...
            schemav200,
            schemav210,
            schemav220alpha
            ...
```

Reply:
@JPinkney WDYT?

Reply:
I left as is. v1 seems even clearer than v100